assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -6,7 +6,7 @@
   style="margin-bottom: 10px;">
   <div ng-transclude flex><!-- expect start time, end time, status details --></div>
   <div row class="text-right text-xs-left">
-    <a href="" ng-click="goChromeless(options)" style="margin-right: 10px;">
+    <a href="" ng-click="goChromeless(options)" ng-if="state!='empty'" style="margin-right: 10px;">
```

Comment:
This causes some flicker when the page loads and there's no log. Maybe `ng-if="state === 'logs'"`?